### PR TITLE
Qoldev 343 Styling underline in alert danger

### PR DESCRIPTION
**Pull request by ghazal-salehi**
https://ssq-qol.atlassian.net/browse/QOLDEV-343

https://oss-uat.clients.squiz.net/contact-us

When you submit contact us form without the required fields:
image

Form io also needs to be updated to remove role='link' from the span. Otherwise the underline will be still there.

### CODE REVIEW

File: src/assets/_project/_blocks/components/link/_qg-link.scss [hunk 1]
```diff
@@ -11,9 +11,8 @@
       .page-link):not(.dfv-cards--type-top-prompt
       a.card):not(.search-categories__index-toggle):not(.search-categories__index):not(.qg-aside-button):not(.dfv-back
       a):not(.QSAR-records-search a):not(.dfv-content a.qg-links-list__link) {
-    &:not(span[role=link] *) {
```
Comment (ThrawnCA) [inline src/assets/_project/_blocks/components/link/_qg-link.scss:14]: This is needed on the Contact Us page. Why was it removed?

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

**Review by ThrawnCA — APPROVED** [review]

**Comment by ghazal-salehi** [comment]
After further discussion with the design team, we decided to keep the underlines. It appears that the comments 'behave' like anchors. Because when you click on the them it will take the user to the causing field using data-component-key values. I'll revert most of the changes made in this PR. And will only increase the text-underline-offset to 5px.

**Comment by ghazal-salehi** [comment]
Bec advised to put this ticket back to backlog until we improve font and font-size overall. I think that would cascade and fix this issue. I'll close this PR and will have a separate one for the logo.

**Comment by ghazal-salehi** [comment]
The pull request was accidentally closed. It will update the underline offset to the links inside the alertbox.

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**